packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -327,9 +347,10 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// Tag for the navigation bar's Hero widget if [transitionBetweenRoutes] is true.
   ///
   /// Defaults to a common tag between all [CupertinoNavigationBar] and
-  /// [CupertinoSliverNavigationBar] instances so they can all transition
-  /// between each other as long as there's only one per route. Use this tag
-  /// override with different tags to have multiple navigation bars per route.
+  /// [CupertinoSliverNavigationBar] instances of the same [Navigator] so they
+  /// can all transition between each other as long as there's only one per
```

Comment:
So long as there's only one what per route?

Reply:
reworded this doc section